docs/servo_power_monitor/src/graph.ts

```diff
@@ -1,26 +1,39 @@
 import Dygraph from 'dygraphs';
 import {Ui} from './ui';
+import {AnnotationText, PowerData} from './power_test_controller';
 
 export class Graph {
   private ui: Ui;
-  private g = new Dygraph('graph', [], {});
+  public g = new Dygraph('graph', [], {});
+  public annotations;
+  public annotationFlag = false;
+  private annotationText: AnnotationText = 'start';
   constructor(ui: Ui) {
     this.ui = ui;
+    this.annotations = this.g.annotations();
   }
-  public updateGraph(powerData: Array<Array<Date | number>>) {
+  public updateGraph(powerData: Array<PowerData>) {
     if (powerData !== undefined && powerData.length > 0) {
       this.ui.hideToolTip();
     }
-    // currentData = data;
     this.g.updateOptions(
       {
         file: powerData,
         labels: ['t', 'ina0'],
         showRoller: true,
+        xlabel: 'Relative Time (s)',
         ylabel: 'Power (mW)',
         legend: 'always',
         showRangeSelector: true,
         connectSeparatedPoints: true,
+        axes: {
+          x: {
+            axisLabelFormatter: function (d) {
+              const relativeTime = (d as number) - powerData[0][0];
+              return (relativeTime / 1000).toLocaleString();
```

Comment:
It would be nice if there is a comment that describes why the time is divided by 1000 (I guess since the time recorded is in ms but the unit of axis is s)